### PR TITLE
WIP:Ft #155388007 flask v1

### PR DESCRIPTION
**Pull request by Rwothoromo**
I have started on the Flask API and would like some guidance.
@matthewacha
@len1
@Thegaijin
@EugeneBad

### CODE REVIEW

**Comment by Rwothoromo** [comment]
Presently, I have been running the apis this way:
`python app/apis/business.py`
Then using postman to test the routes
The data used was not being retrieved/created from the class objects

**Comment by matthewacha** [comment]
Did it run on postman?? you are on track, just polish up the logic and write tests for the endpoints

**Comment by Rwothoromo** [comment]
@matthewacha it is running on postman, my issue is more with how are things looking and any technical advice

File: app/apis/auth.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+from flask import Flask, jsonify
+from flask_restful import Resource, Api
+from flask_restful.reqparse import RequestParser
+
+app = Flask(__name__)               # Create Flask WSGI appliction
+api_v1 = Api(app, prefix="/api/v1")  # Wrap the app in Api
+
+# users list of user dictionary objects
+users = []
```
Comment (Thegaijin) [inline app/apis/auth.py]: When you have to pick a user from the list, how will you go about it?

Comment (Rwothoromo) [reply app/apis/auth.py]: I added a get_user function that checks if the user exists

**Pull request merged**